sgv2-docsapi/src/test/java/io/stargate/sgv2/docsapi/testprofiles/MaxDepth4.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package io.stargate.sgv2.docsapi.testprofiles;
+
+import io.quarkus.test.junit.QuarkusTestProfile;
+import java.util.Map;
+
+public class MaxDepth4 implements QuarkusTestProfile {
```

Comment:
Can we call this `MaxDepth4TestProfile` and `MaxDepth8TestProfile` since all the others test profiles also have the `TestProfile` suffix?